src/python/geoclaw/surge/storm.py

```diff
@@ -1498,6 +1498,39 @@ def available_models():
     return output
 
 
+def make_multi_structure(path):
+    # create a dictionary of Storm objects for ATCF files with multiple storm tracks in them
+    fileRead = open(path, 'r')
+    lines = fileRead.readlines()
```

Comment:
The cleaner way to write this piece of code is probably with a `with` (that sounds redundant, I know)